agents/scripts/install_deps_and_build.sh

```diff
@@ -57,6 +57,9 @@ install_python_requirements() {
       fi
     done
   fi
+
+  echo "pre-import python modules..."
+  python3.10 -c "import llama_index.core;"
```

Comment:
@plutoless It's a reasonable workaround to avoid the unexpected downloading during run. I'm ok with it at the moment, add some comment to describe the issue might be better. However, for long term, may be we can have a pre/post install scripts section in each extension's manifest.json, which may be triggered by tman install? @Halajohn